### DISCUSSION TITLE
How do I extract the changepoints from the forecast?

### DISCUSSION THREAD

**Discussion by horvaj6066** [ourownstory/neural_prophet#1092] [Q&A - get help using NeuralProphet]
I'm just trying to calculate the trend growth between the NeuralProphet selected changepoints. In Prophet it was just m.changepoints but I can't seem to extract them from NeuralProphet. Does anyone know how I go about extracting this list of changepoints?  
thanks, 
Jon

**Comment by Biebrya** [comment]
I am also wondering where this is located, in Prophet docs for example they have a add_changepoints_to_plot() function, this would be a nice feature to have in neuralprophet as well.

**Comment by judussoari** [comment]
Thanks! We will discuss it in the dev core team :)

**Comment by horvaj6066** [comment]
I figured it out actually but thanks for responding.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
